yoga/Yoga.h

```diff
@@ -235,6 +235,10 @@ WIN_EXPORT void YGConfigSetExperimentalFeatureEnabled(const YGConfigRef config,
 WIN_EXPORT bool YGConfigIsExperimentalFeatureEnabled(const YGConfigRef config,
                                                      const YGExperimentalFeature feature);
 
+WIN_EXPORT void YGConfigSetUseWebDefaults(const YGConfigRef config, const bool enabled);
```

Comment:
Can you add a comment here expressing that web defaults is the preferred configuration and non-web defaults is to be considered legacy?